src/BaseApi.js

```diff
@@ -65,6 +65,10 @@ const getHeaders = async (isFormData = false) => {
   if (token) {
     headers['Authorization'] = 'Token ' + token
   }
+  const lang = await AsyncStorage.getItem('language')
+  if (lang && lang == 'de') {
+    headers['Accept-Language'] = lang
+  }
```

Comment:
We need to send the accept-language header to get the right translation from backend, eg for dates. To avoid to receive eg russian dates, we only send header if language is German.